tools/python/liveanalytics_migration_scripts/test_scripts/influxdb-restart.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+#
+# The MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+set -e
+
+DEFAULT_INFLUXDB_V2_VERSION="latest"
+INFLUXDB_V2_VERSION="${INFLUXDB_V2_VERSION:-$DEFAULT_INFLUXDB_V2_VERSION}"
+INFLUXDB_V2_IMAGE=influxdb:${INFLUXDB_V2_VERSION}
+
+SCRIPT_PATH="$( cd "$(dirname "$0")" ; pwd -P )"
+
+CONTAINER_NAME=migration_test_influxdb_v2
+NETWORK_NAME=migration_test_influx_network
+
+docker kill $CONTAINER_NAME || true
+docker rm $CONTAINER_NAME || true
+docker network rm $NETWORK_NAME || true
+docker network create -d bridge $NETWORK_NAME --subnet 192.168.0.0/24 --gateway 192.168.0.1
```

Comment:
```suggestion
docker network create -d bridge $NETWORK_NAME
```

We should avoid hard coding the subnet and let docker decide to avoid a potential IP conflict